### PR TITLE
Walls/windows Resprite

### PR DESCRIPTION
**Pull request by FoxxoTrystan**
## About the PR

Changes Solid Wall / Reinforced Wall and all windows sprties to a more modern and unique look and texture.
Windows will now blend with the walls, making it look **PERFECT!**

Atmos Vents got updated with norot value to make sure they always face front.

Added Plastitanium Window.
Changed Plating (Normal Plating and Asteroid Plating) Sprite.
Changed Shuttle Floor Sprties.

Makes Indestructible Plastitanium Walls not whitelsited from RCD.

## Why / Balance

Walls/Windows are from the old TG Days (even tho they still uses it), those sprites are more modern and feel more refreshed and polished then the old one, its now blend with the walls/windows, look less like a mess and well in general looks good.

## Technical details

Changes IconSmoth for windows and just edited sprites files, also added a new window.

## Media

image
image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: FoxxoTrystan
- add: Plastitanium Windows
- fix: Vents now always face Front.
- tweak: Changed Solid/Reinforced walls sprites.
- tweak: Changed all windows sprites.
- tweak: Windows will now blend with walls.
- tweak: Changed plating sprite.
- tweak: Changed Shuttle Floor sprite.

### CODE REVIEW

File: Resources/Textures/Structures/Doors/secret_door.rsi/meta.json
Comment (Cheackraze) [inline Resources/Textures/Structures/Doors/secret_door.rsi/meta.json:1]: make sure the copyright info is being updated properly

File: Resources/Textures/Structures/Windows/plastitanium_window.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from Paradise",
```
Comment (Cheackraze) [inline Resources/Textures/Structures/Windows/plastitanium_window.rsi/meta.json:4]: cc-by-sa-3 requires us to be more specific. If you could point to the repository directly would suffice.

File: Resources/Textures/Tiles/plating.png
Comment (Cheackraze) [inline Resources/Textures/Tiles/plating.png:1]: bold, trying to slip in some tile changes. However, I definitely approve of the lower contrast ones here.

**Review by Cheackraze — CHANGES_REQUESTED** [review]

File: Resources/Textures/Tiles/plating.png
Comment (FoxxoTrystan) [reply Resources/Textures/Tiles/plating.png:1]: Tile changed is noticied tho i forgot to say that the shuttle floors was also changed.

File: Resources/Textures/Structures/Windows/plastitanium_window.rsi/meta.json (hunk 1 above)
Comment (FoxxoTrystan) [reply Resources/Textures/Structures/Windows/plastitanium_window.rsi/meta.json]: Fixed same with tiles attribution.

File: Resources/Textures/Tiles/plating.png
Comment (FoxxoTrystan) [reply Resources/Textures/Tiles/plating.png:1]: Shuttle Floor changes is now listed in Changlog and "about this PR" same for the Indestructible Plastitanium RCD Remove thing.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**